solveengine/mipmodel.py

```diff
@@ -220,7 +254,7 @@ def set_to_maximize(self):
         self.set_direction(Direction.MAXIMIZE)
     
     def build_with_matrices(self, f, A, b, 
-                            Aeq = None, beq = [], 
+                            Aeq=None, beq=[],
                             lb=[], ub=[], 
                             int_list=[], bin_list=[]):
```

Comment:
Wow, did not see thats, thats interesting.. Will figure out a solution (tuple or maybe a None default value)